demo/package.json

```diff
@@ -15,8 +15,8 @@
     "@react-navigation/bottom-tabs": "^7.2.0",
     "@react-navigation/native": "^7.0.14",
     "raygun4reactnative": "file:../sdk",
-    "react": "^19.0.0",
```

Comment:
why did we need to pin this version?

Reply:
Both version changes are related. I had an error telling me that I could not use `react` `19.1.0` and `react-native-renderer` `19.0.0`, the versions had to match, so I changed them all to match. Probably the version was updated automatically through dependabot.

Reply:
thanks just checking